betty/extension/webpack/assets/working/webpack.config.js

```diff
@@ -0,0 +1,159 @@
+'use strict'
+
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin')
+const path = require('path')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const TerserPlugin = require('terser-webpack-plugin')
+const configuration = require('./webpack.config.json')
+const webpack = require('webpack')
+
+class EntryScriptCollector {
```

Comment:
Document this class. Can we make our JSlinting stricter (in a separate PR?) to require docblocks?